SUSE/scf
# Bump UUA to revision using PXC.

## Description

https://jira.suse.com/browse/CAP-260

Subordinate PR: https://github.com/SUSE/uaa-fissile-release/pull/159

## Test plan

  - Build a cluster.
  - Start it.
  - See all of UAA running in the green.
  - Test with :smoking:, brain, :cat: 

Note, on Vagrant :cat: may have failures pulled in from head of dev.
